linux_os/guide/system/software/integrity/software-integrity/aide/aide_periodic_cron_checking/oval/shared.xml

```diff
@@ -53,4 +58,36 @@
     <ind:instance datatype="int" operation="greater than or equal">1</ind:instance>
   </ind:textfilecontent54_object>
 
+  <linux:systemdunitproperty_object id="obj_aidecheck-service_unitfilestate" version="1">
+    <linux:unit>aidecheck.service</linux:unit>
+    <linux:property>UnitFileState</linux:property>
+  </linux:systemdunitproperty_object>
+  <linux:systemdunitproperty_object id="obj_aidecheck-timer_unitfilestate" version="1">
+    <linux:unit>aidecheck.timer</linux:unit>
+    <linux:property>UnitFileState</linux:property>
+  </linux:systemdunitproperty_object>
+  <linux:systemdunitproperty_object id="obj_aidecheck-timer_activestate" version="1">
+    <linux:unit>aidecheck.timer</linux:unit>
+    <linux:property>ActiveState</linux:property>
+  </linux:systemdunitproperty_object>
+  <linux:systemdunitproperty_state id="ste_aide_is_enabled" version="1">
+    <linux:value>enabled</linux:value>
+  </linux:systemdunitproperty_state>
+  <linux:systemdunitproperty_state id="ste_aide_is_active" version="1">
+    <linux:value>active</linux:value>
+  </linux:systemdunitproperty_state>
+
+    <linux:systemdunitproperty_test check="all" id="tst_aidecheck-service_enabled" version="1" comment="systemd aidecheck.service enabled">
```

Comment:
Could you adjust these lines to a length of 99, please?

Reply:
Done in [5c9f042](https://github.com/ComplianceAsCode/content/pull/10589/commits/5c9f04285a2293a8d6b080d598bfae92bdea872e)